### PR TITLE
Feature: adding new driver for Agilent signal generator E8267C

### PR DESCRIPTION
**Pull request by euchas**
added all basic commands. 

why no one likes 💃 :(
@giulioungaretti @jenshnielsen

### CODE REVIEW

File: qcodes/instrument_drivers/agilent/E8267C.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        self.add_parameter('pulse_width',
+                           label='Pulse width',
+                           unit='ns',
+                           set_cmd='PULM:INT:PWID {}',
+                           get_cmd='PULM:INT:PWID?',
+                           vals=Numbers(min_value=10e-9,
+                                        max_value=20e-9))
+        self.add_parameter(name='phase',
+                           label='Phase',
+                           unit='rad',
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/agilent/E8267C.py]: looks like this is degrees?

File: qcodes/instrument_drivers/agilent/E8267C.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                                        'ON': 1})
+# reset values after each reconnect
+        self.power(0)
+        self.power_offset(0)
+        self.connect_message()
+        self.add_function('reset', call_cmd='*RST')
+
+# functions to convert between rad and deg
+
+    def deg_to_rad(self, angle_deg):
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/agilent/E8267C.py]: These functions are in numpy https://docs.scipy.org/doc/numpy/reference/generated/numpy.deg2rad.html perhaps use them from there

Comment (giulioungaretti) [inline qcodes/instrument_drivers/agilent/E8267C.py]: definitely use numpy !

**Comment by euchas** [comment]
@jenshnielsen @giulioungaretti fixed the numpy functions

**Pull request merged**